conans/client/command.py

```diff
@@ -1174,6 +1185,17 @@ def _add_common_install_arguments(parser, build_help):
     if build_help:
         parser.add_argument("--build", "-b", action=Extender, nargs="*", help=build_help)
 
+def _get_option_string(parser, option):
+    """ Returns a string with all the option names in the parser for the 'option'.
+    e.g "--install-folder/--install_folder/-if"
+
+    :param parser: parser to extract the option from
+    :param option: String with parser option. e.g. "--install_folder"
+    :return: formatted string with all option names like ""--option_name/--option-name/-on"
+    """
+
+    return '/'.join(parser._get_option_tuples(option)[0][0].option_strings)
```

Comment:
Using a private memeber of parser, is a smell. We might do it as an exceptional thing in our code, but not to third parties

Reply:
Ok, will try see if there is another approach to reach that information inside parser